modules/core/05-port/types/module.go

```diff
@@ -90,19 +90,22 @@ type IBCModule interface {
 		ctx sdk.Context,
 		packet channeltypes.Packet,
 		relayer sdk.AccAddress,
+		channelVersion string,
```

Comment:
I feel like the ordering could be: ctx, channelVersion, packet, relayer, but fine with how it is

Reply:
Yeah, it would be nicer, maybe. This is the chance to change it since we are breaking the API anyway :) I can change it if people agree